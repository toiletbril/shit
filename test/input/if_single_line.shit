if (if (1 == 1) then 27 else 54 fi) == 54 then 4 else 10 fi
