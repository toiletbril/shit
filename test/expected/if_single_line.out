[If]
  [Binary ==]
    [If]
      [Binary ==]
        [Number 1]
        [Number 1]
      [Number 27]
    [Else]
      [Number 54]
    [Number 54]
  [Number 4]
[Else]
  [Number 10]
10
